### DISCUSSION TITLE
How to put link in Spoke to arbitrary web content?

### DISCUSSION THREAD

**Discussion by davegoopot** [mozilla/hubs#2981] [General Questions & Discussion] [answered]
I'd like to put an element in my Spoke scene that when clicked navigates to an arbitrary webpage. Is that possible?

I've seen that you can add a link element to another Hub. If instead of a Hub link I put in https://example.com/, then in the Hub it displays an "Unable to load Video" message. If you "open link" on that message, it navigates to https://example.com/.

Is there a way to link using an image element, say?

**Comment by brianpeiris** [comment, marked as answer]
This sounds like a duplicate of #2979. Please continue the discussion there.
